tomviz/LoadDataReaction.h

```diff
@@ -16,13 +16,17 @@
 #ifndef tomvizLoadDataReaction_h
 #define tomvizLoadDataReaction_h
 
+#include <vector>
```

Comment:
I think you should use a QList here, and should just declare the type here, include its header in the implementation.

Reply:
Fixed in https://github.com/OpenChemistry/tomviz/pull/1535/commits/aac9bca28336d6a2d404826c0e0b69b8294ed33a